### PR TITLE
Fix if folder is not on PATH

### PR DESCRIPTION
**Pull request by seanshpark**
- add script path case not in PATH
- add 'TEST_FILE' as an option

### CODE REVIEW

File: dotnet-runtest.sh [hunk 1]
```diff
@@ -67,7 +67,12 @@ if [ ! -d $TEST_ROOT ]; then
 	TEST_ARCHIVE=~/${TEST_SET}.${HASH}.zip
 
 	if [ ! -e $TEST_ARCHIVE ]; then
-		TEST_ARCHIVE=~/$(ls -t ~ | grep $TEST_SET | head -1)
+		TEST_FILE=$(ls -t ~ | grep $TEST_SET | head -1)
+		if [ -z "$TEST_FILE" ]; then
+			TEST_ARCHIVE=$BASE_PATH/${TEST_SET}.zip
+		else
+			TEST_ARCHIVE=~/${TEST_FILE}
+		fi
```
Comment (lemmaa) [inline dotnet-runtest.sh]: How about change the search order? Searching current directory first is general approach, I think :)

File: dotnet-build-arm.sh [hunk 2]
```diff
@@ -317,7 +327,7 @@ then
         rm -rf $BASE_PATH/coreclr/bin/tests/$OVERLAY
     fi
     message "[BUILD CORECLR TEST PACKAGE]"
-    dotnet-runtest.sh Linux.${ARCHITECTURE}.$CAP_CONFIGURATION --build-overlay-only
+    $__Bash_dir/dotnet-runtest.sh Linux.${ARCHITECTURE}.$CAP_CONFIGURATION
 fi
```
Comment (lemmaa) [inline dotnet-build-arm.sh]: Do we need to assume that `dotnet-runtest.sh` and this script must be in same path?

File: dotnet-runtest.sh (hunk 1 above)
Comment (seanshpark) [inline dotnet-runtest.sh]: OK

File: dotnet-build-arm.sh (hunk 2 above)
Comment (seanshpark) [inline dotnet-build-arm.sh]: adding `$__Bash_dir/` is for running `dotnet-runtest.sh` file if the folder is not in the `PATH` environment. I didn't try with adding `./` though.

> Do we need to assume that `dotnet-runtest.sh` and this script must be in same path?

I think I assumed `dotnet-build-arm.sh` and `dotnet-runtest.sh` be in the same folder as current files are so.

File: dotnet-build-arm.sh [hunk 3]
```diff
@@ -317,7 +327,7 @@ then
         rm -rf $BASE_PATH/coreclr/bin/tests/$OVERLAY
     fi
     message "[BUILD CORECLR TEST PACKAGE]"
-    dotnet-runtest.sh Linux.${ARCHITECTURE}.$CAP_CONFIGURATION --build-overlay-only
+    $__Bash_dir/dotnet-runtest.sh Linux.${ARCHITECTURE}.$CAP_CONFIGURATION
```
Comment (lemmaa) [inline dotnet-build-arm.sh]: You need to keep `--build-overlay-only` option. This option must passed to `runtest.sh` via `$EXTRA_OPTION` in `dotnet-runtest.sh`. Sorry for lack of comment and documentations. :(

Please reference the PR https://github.com/dotnet/coreclr/pull/6807 in review for this option.

**Comment by seanshpark** [comment]
Changed search order for the first comment (please point me if not what you think). I'll push another change after after what should be done.

File: dotnet-build-arm.sh (hunk 3 above)
Comment (seanshpark) [inline dotnet-build-arm.sh]: oops. OK.

**Comment by lemmaa** [comment]
LGTM. Thank you.

**Pull request merged**